src/bin/travis/resources/centaur.wdl

```diff
@@ -22,7 +22,7 @@ task centaur {
 
     runtime {
         docker: "geoffjentry/centaur-cromwell:latest" 
```

Comment:
Is there a Tech Debt ⚓ ticket to have this image somewhere not under your personal Docker account?